gymnasium/gymstatic
# GYM-016 Meta Update

## What this PR does:

- Adds temp layout for GYM-016 meta
- Adds GYM-016 front matter
- Adds GYM-016 artwork

### To Test:

- View source: https://deploy-preview-458--thegymcms.netlify.app/static/courses/meta/gym-016-meta/
- Verify: `<meta name="twitter:card" content="summary_large_image">`
